documentation/source/overview/studies.rst

```diff
@@ -33,6 +33,7 @@ The following studies have used SCT:
 2020
 ----
 
```

Comment:
This is a minor, unrelated thing, but I've just noticed the second 2021 reference is missing italics.

![Screenshot from 2021-09-15 15-57-39](https://user-images.githubusercontent.com/16181459/133500905-51c4a130-93ab-4287-b4cc-954e6a7bde5e.png)

I can sneak a fix for this into this PR while we're at it.

Reply:
==> ab7c4a5